duckduckgo/macos-browser
# Update dark mode shield icon asset

Task/Issue URL: https://app.asana.com/0/0/1201318495713876/1201493924825293/f
Tech Design URL:
CC:

**Description**:
Updates dark mode shield icon asset

Before | After
--- | ---
<img width="650" alt="Screenshot 2021-12-14 at 12 02 08" src="https://user-images.githubusercontent.com/635903/146182644-2433977d-517b-4385-bd1d-d0da61c429ff.png"> | <img width="648" alt="Screenshot 2021-12-15 at 11 53 14" src="https://user-images.githubusercontent.com/635903/146182660-d67b5bbf-128f-498f-abf8-7f891c44bbd0.png">
<img width="794" alt="Screenshot 2021-12-15 at 11 56 21" src="https://user-images.githubusercontent.com/635903/146182696-22cf4490-a389-4b5b-aa4f-0c4e9345f8bb.png"> | <img width="794" alt="Screenshot 2021-12-15 at 11 55 53" src="https://user-images.githubusercontent.com/635903/146182705-9f4036e1-43fb-4126-b2b3-8a45799a7119.png">


**Steps to test this PR**:
1. Set up system to use dark theme
2. Go to https://google.com
3. Confirm red dot on the shield icon doesn't have a white border

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
